Do not statically link with Curl and OpenSSL.  This also removes
the openssl-src dependency.

--- Cargo.toml.orig	2021-06-08 17:26:12 UTC
+++ Cargo.toml
@@ -15,7 +15,7 @@ chardet = "0.2.4"
 chrono = { version = "0.4.11", features = ["serde"] }
 clap = { version = "2.33.1", default-features = false, features = ["suggestions", "wrap_help"] }
 console = "0.14.1"
-curl = { version = "0.4.38", features = ["static-curl", "static-ssl"] }
+curl = { version = "0.4.38" }
 dirs = "3.0.2"
 dotenv = "0.15.0"
 elementtree = "0.5.0"
